README.md

```diff
@@ -26,39 +40,32 @@ pip install -r requirements.txt
-* Aligning the TSP documented that way with trace-server's own (reference) endpoints is a work in progress.
+### Generate the API
 
-In the meantime, Swagger has recently been added to trace-server.
+Swagger has recently been added to the Trace Compass trace-server (reference implementation).
 
-* Below is how to generate the alternate TSP version, according to Swagger in trace-server.
-* This alternate version is to be gradually augmented based on what has been manually documented through `API.yaml`.
-* [Swagger github pages][7] show this alternate version to grow.
-* `API.yaml` should gradually become the wanted version of the TSP, as this alternate one shows the reference TSP.
-* Such reference and future (forecasted) TSP versions will be documented accordingly as these yaml files evolve.
+* Below is how to generate the TSP version, according to Swagger in trace-server.
+* The generated TSP should match the current supported version of the TSP. Any differences may be pushed for review.
+* `API.yaml` shows now the current supported version of the TSP.
 
-1. Import all [TC][2] and [incubator][3] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
-1. Run trace-server or project [1] below as Eclipse Application configured this way (or similar):
-   * `-Xms512m -Xmx1g`.
-   * Launch with `all workspace and enabled target plug-ins`.
-1. Browse [to here][4] ([swagger][5]) or so to generate server's TSP.
-1. Bring the resulting file over; e.g.: `mv ~/Downloads/openapi.yaml .`
+1. Import all [TC][tracecompass] and [incubator][incubator] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
+1. Open `traceserver.product` file in plug-in `org.eclipse.tracecompass.incubator.trace.server.product`.
+1. Click on the `Run` button on the top right corner of the opened `traceserver.product`.
+1. Browse [to here][apiyaml] ([swagger][swagger]) or so to generate server's TSP.
+1. The resulting file is stored in the user's Downloads directory; e.g.: `~/Downloads/openapi.yaml`
+1. Copy `~/Downloads/openapi.yaml` to this directory.
 1. Update the latter with its license information and remove extra information: `./openapi.py`
-1. The resulting git diff may then be pushed for review, at will.
-
-[1] org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core
-
-[2]: https://projects.eclipse.org/projects/tools.tracecompass/developer
-[3]: https://projects.eclipse.org/projects/tools.tracecompass.incubator/developer
-[4]: http://localhost:8080/tsp/api/openapi.yaml
-[5]: https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Integration-and-configuration#openapiresource
-[6]: https://theia-ide.github.io/trace-server-protocol/
-[7]: https://theia-ide.github.io/trace-server-protocol/swagger/
-[8]: https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi
+1. The resulting diff between `API.yaml` and `openapi.yaml` can then be pushed for review. 
```

Comment:
Trailing space, to remove; `markdownlint` tells you so in VS Code.

Reply:
ok